lib/node_modules/@stdlib/math/base/ops/README.md

```diff
@@ -87,10 +87,37 @@ The namespace contains the following functions:
 <!-- eslint no-undef: "error" -->
 
 ```javascript
+var Complex128 = require( '@stdlib/complex/float64' );
 var objectKeys = require( '@stdlib/utils/keys' );
 var ns = require( '@stdlib/math/base/ops' );
 
 console.log( objectKeys( ns ) );
```

Comment:
Let's not print out the namespace contents anymore; that was mostly a placeholder:

```suggestion
var Complex128 = require( '@stdlib/complex/float64' );
var ns = require( '@stdlib/math/base/ops' );
```

Users can always browse the docs to see what packages are part of the namespace.